lib/index.js

```diff
@@ -25,15 +27,28 @@ function initServer() {
     return server;
 }
 
-function initLogger() {
+function initLogger(abbreviation) {
     const bunyan = require('bunyan');
+    // initialize maximum body logging length
+    bodyAbbreviationLength = abbreviation;
     return bunyan.createLogger({
         name: packageJson.name
     });
 }
 
 function logJson(logger, body) {
-    logger.info(JSON.stringify(body, null, 4));
+    // Make a copy so the original body doesn't get mutated
+    const content = Object.assign({}, body);
+    // If the body is too long of a string, abbreviate it
+    if (bodyAbbreviationLength > 10 && content.body && content.body.length && content.body.length > bodyAbbreviationLength) {
+        const halfAbbrev = bodyAbbreviationLength / 2;
+        const secondHalfStart = content.body.length - halfAbbrev + 5;
```

Comment:
`String.prototype.slice()` supports negative indexing, so the second half can just use `-halfAbbrev`.  Is the `+ 5` to compensate for the elipsis?  That would seem fine but not necessary to me.

Reply:
My thought process for how this calculation works is that the end result of the body length would be the exact length you specified with `--abbrev` including the ellipsis, so `<half the length><5 char ellipsis><half the length -5>`. The min length of the abbreviation being 11 was so that the end result wouldn't end weird on the ellipsis, because if the length is less than 2x the ellipsis length then you end up with a string that's actually longer than the original. The whole calculation is honestly sort of arbitrary, realistically the output length is only an issue once it gets above around 500 characters.